app/assets/stylesheets/partials/_click_to_play.scss

```diff
@@ -10,6 +10,8 @@
     .img-container {
       img {
         width : 100%;
+        height: 100%;
+        object-fit: contain;
```

Comment:
I never heard of this property until now :) It doesn't look like it is supported in IE though.